AUTHORS

```diff
@@ -9,4 +9,13 @@ Genjix - Porting pro-mode functionality to lite-gui and worked on server
 Slush - Work on the server. Designed the original Stratum spec.
 Julian Toash (Tuxavant) - Various fixes to the client.
 rdymac - Website and translations.
-kyuupichan - Miscellaneous.
\ No newline at end of file
+kyuupichan - Miscellaneous.
+Sombernight = many fixes
+Jonald Fyookball - Imperator
+Calin Culiannu - iOS version, MacOS packaging, MacOS QRScanner, miscellaneous
+Malcolm Smith - New Android Version, tuneups and fixups
+Roger Taylor - new version 1.4 blockchain synch code, external plugin system, miscellanous
+Mark Lubdeberg - Windows zbar, Coin Token work, various miscellanity
```

Comment:
Lund?

Reply:
Lol. Typo. 

Anything else you want to appear next to your name? Can be anything even a joke.

Reply:
I’ll fix when I get home